content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,69 @@
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
+# INTRODUCTION TO RELATIONAL DATABASE 
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+**LESSONS**
 
+- Lesson 1: 
+  > Data and Tables:
+    * learn the principles behind relational data organization:tables,queries,aggregations,
+    keys and joins.
+- Lesson2:
+    > Elements of **SQL**
+    * start learning  by using **select** & **insert** statements/commands to read and write
+     data to database tables
+- Lesson3: 
+  > Nodejs SQL-API: 
+    * learn SQL-APIs and  apply your knowledge to fix common bugs that arise from database-backend-web service
+  > Python SQL-API: 
+    * learn SQL-APIs and  apply your knowledge to fix common bugs that arise from database-backend-web service
+
+- Lesson4:
+  > Deeper Into SQL 
+## What a Relational Database Is:
+A relational database is a type of database that stores and provides access to data points that
+are related to one another. Relational databases are based on the relational model, an intuitive,
+straightforward way of representing data in tables. In a relational database, each row in the table
+is a record with a unique ID called the key. The columns of the table hold attributes of the data,
```

Comment:
id fields aren't compulsory. Just good practice. 

And if someone doesn't know what sql is about then this paragraph won't help them at all.  Either link to a nice explanation or ask Kii to help you reword this. 

I also think you should be more actively involved with the Thing Explainer program.